Gemfile.lock

```diff
@@ -859,6 +864,7 @@ DEPENDENCIES
   virtus (~> 1.0.5)
   web-console (~> 2.0)
   webmock (~> 3.1.1)
+  webrick (~> 1.3, >= 1.3.1)
```

Comment:
@HalahAb , what does adding `webrick` accomplish? (I'm not opposed to it; I just want to make sure we have a good reason to add new gems)

Reply:
I was prompted to install it after upgrading to capybara 3.3.1 I think its running the capybara.server, but I've since reverted back to version 2.18. I couldn't reconcile the differences that lead to the test breaks, and I didn't want to take up too much time trying. 2.18 seems to be working fine for now.